docs/internships/outreachy/contribution.md

```diff
@@ -0,0 +1,32 @@
+---
+id: contribution-process
+title: Contribution Process
+sidebar_label: Contribution Process
+---
+
+### Introduction
+
+The Palisadoes Foundattion has a sponsored work on Talawa projects for some years now. Talawa is basically an open-source project to help community-based organizations such as clubs, neighborhood groups,religious institutions etc manage their daily activities.
+The Talawa applications has 3 components
+1. Talawa-API
+1. Talawa
+1. Talawa-Admin
+
+You can learn more about Palisodes Foundation and Talawa projects **[here](docs/about.md)** or in the FAQ and it's [core concepts](docs/applications/overview.md). You can learn about Talawa  [applications](docs/applications/architecture.md). 
+
+Check our project [repositories](docs/internships/outreachy/getting-started.md) for Talawa software components listed above.
+
+To know more about our community join our [slack channel](https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-1qldywc0i-9pEHNIG2B38JhsAbRSbuug).
+
+
+### How to Contribute
+
+To contribute to the our projects,
+
+1. Fork the repository by clicking on the fork button on the repository. This creates a copy of the repository to the application github account
+1. Clone the forked (copied) repository
+1. Follow the installation guide and install required depencies
+1. Go through the project and open an issue if you notice a bug pr feature you would want to be added.
+1. Once the issue is assigned to you, 
+1. Create a new branch and make your changes to the project 
+1. Open a pull request(PR)
```

Comment:
Add a closing sentence after these and any other bullets in the PR. It makes the flow better.